### PR TITLE
Toggle workspace switch popup

### PR DESCRIPTION
**Pull request by l3nn4rt**
This PR adds a toggle to show/hide the workspace switch popup when scrolling on the panel to change workspace.

### CODE REVIEW

**Comment by charlesg99** [comment]
Hello l3nn4rt, thank you for your contribution. The first thing to do here is to remove all po modifications, aside from maybe the Italian one. Ideally, you could remove it as well as there is additional translation that'll need to be done before we release v38, so you could help with it then. Also, to reuse the gnome-shell code as much as possible, to prevent the popup from appearing I would just do something like this:

**Comment by l3nn4rt** [comment]
Alright, I force-pushed directly to this PR the solution you suggested.
It seems to work as expected for me.

File: Settings.ui [hunk 1]
```diff
@@ -3235,6 +3235,69 @@
             </child>
           </object>
         </child>
+        <child>
+          <object class="GtkListBoxRow" id="listboxrow_scroll_panel_options_show_ws_popup">
+            <property name="width_request">100</property>
+            <property name="height_request">80</property>
```
Comment (charlesg99) [inline Settings.ui:3241]: Please remove the size requests so the row has the same size as the one above it:

image

File: panel.js [hunk 2]
```diff
@@ -1249,7 +1249,11 @@ var dtpPanel = Utils.defineClass({
                 //gnome-shell < 3.30 needs an additional "screen" param
                 global.screen ? args.push(global.screen) : 0;
 
+                if (!Me.settings.get_boolean('scroll-panel-show-ws-popup'))
+                    Main.wm._workspaceSwitcherPopup = { display: () => {} };
                 Main.wm._showWorkspaceSwitcher.apply(Main.wm, args.concat([0, { get_name: () => 'switch---' + direction }]));
+                if (!Me.settings.get_boolean('scroll-panel-show-ws-popup'))
```
Comment (charlesg99) [inline panel.js:1255]: You can remove this `if` and just set `Main.wm._workspaceSwitcherPopup = null;` even if we are showing the workspaces popup.

File: Settings.ui (hunk 1 above)
Comment (l3nn4rt) [reply Settings.ui]: :heavy_check_mark: done

File: panel.js (hunk 2 above)
Comment (l3nn4rt) [reply panel.js:1255]: This wouldn't prevent multiple instances of `WorkspaceSwitcherPopup` from being spawned, on multiple scrolls.
Low quality GIF, buy you should be able to notice transparency variation here:
set popup to null without checking flag

Comment (charlesg99) [reply panel.js:1255]: Ha yes you're absolutely right... I just don't like the 2 consecutive `Me.settings.get_boolean('scroll-panel-show-ws-popup')` calls, can we set this boolean in a variable or something?

Comment (l3nn4rt) [reply panel.js]: :heavy_check_mark: done

**Comment by charlesg99** [comment]
Looks great to me! 

@jderose9 good for you as well?

**Comment by jderose9** [comment]
Thanks so much, @l3nn4rt !!

**Comment by l3nn4rt** [comment]
Thanks to @charlesg99 for kindly reviewing the changes :)

**Pull request merged**